internal/provider/azure/azure.go

```diff
@@ -500,6 +499,12 @@ func FindDefaultGeoTarget(endpoints []*externaldnsendpoint.Endpoint) string {
 	return ""
 }
 
+func GenerateProfileName(resourceGroup string, ep *externaldnsendpoint.Endpoint) string {
+	data := []byte(ep.DNSName)
```

Comment:
Is DNSName unique enough? Multiple endpoints can have the same DNSName depending on how the provider works.

Reply:
As DNS name is the host of the CNAME that will point to the traffic manager profile, I'd have thought if that same hostname shows up again somewhere else, it would need to resolve to the same traffic manager profile?

Reply:
true, so this is a name that we want linked to the dnsname i guess